website/oauth/views.py

```diff
@@ -41,17 +50,30 @@ def oauth_connect(service_name, auth):
 
 @must_be_logged_in
 def oauth_callback(service_name, auth):
-    user = auth.user
-    provider = get_service(service_name)
+    if waffle.flag_is_active(request, features.ENABLE_GV):
+        _forward_to_addon_service()
```

Comment:
super nit: adding a `return {}` here lets you unindent the rest of the function, reducing the (apparent) cyclomatic complexity